kore/src/Kore/Equation/Application.hs

```diff
@@ -228,7 +228,7 @@ attemptEquation sideCondition termLike equation =
             let toMatchResult Conditional { predicate, substitution } =
                     (predicate, Substitution.toMap substitution)
             Substitution.mergePredicatesAndSubstitutions
-                sideCondition
+                SideCondition.top
```

Comment:
I assumed it would be fine to not include the side condition here, since we use it in `checkRequires`. Is this correct or am I overlooking something?